packages/docs/src/styles/_examples.scss

```diff
@@ -192,6 +192,13 @@ $options-margin: $pt-grid-size * 3;
   }
 }
 
+.docs-skeleton-example {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  width: 100%;
```

Comment:
remove this class per https://github.com/palantir/blueprint/pull/338/files/4d90002dbe0139462692f1abd52eef6b75b0a07c#r91629587